decidim-core/app/packs/src/decidim/geocoding.js

```diff
@@ -48,7 +24,9 @@ $(() => {
         $input.trigger("geocoder-suggest-coordinates.decidim", [selectedItem.coordinates]);
       }
     });
+  });
 
-    $input.data("geocoder-tribute", tribute);
+  $("[data-decidim-geocoding]").on("open close", (event) => {
```

Comment:
I think this should be moved to the base `AutoComplete` class in order for the same fix to apply everywhere autocomplete is used.